ui/pages/swaps/swaps.util.js

```diff
@@ -79,6 +79,8 @@ const getBaseUrlForNewSwapsApi = (type, chainId) => {
 };
 
 export const getBaseApi = function (type, chainId = MAINNET_CHAIN_ID) {
+  // eslint-disable-next-line no-param-reassign
+  chainId = chainId === RINKEBY_CHAIN_ID ? MAINNET_CHAIN_ID : chainId;
```

Comment:
Not sure is this ok to be committed to develop ?

Reply:
Yup, we already have this code in the `smart-transactions` branch, but this will get merged faster. The reason is that our APIs don't support Rinkeby, but with this change we get data for mainnet, which we can use for Rinkeby in Swaps.